modules/moderation.py

```diff
@@ -604,7 +604,7 @@ async def _strike_set(self, ctx, member: discord.Member, count: StrikeRange, *,
                 return await ctx.message.delete()
 
             await ctx.send(
-                f'{member} ({member.id}) has been destuken, '
+                f'{member} ({member.id}) has had strikes removed, '
```

Comment:
Still a bit awkward, instead of reiterating how much they have left later on lets just add it here "... had x strikes removed" or similar